examples/tutorials/translations/hindi/Part 05 - Welcome to the Sandbox.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>English version of notebook doesn't contain any author names. Did you check the history for it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Will add github link of my profile here :)</p><p> </p>